spec/eofv0_verkle.md

```diff
@@ -0,0 +1,107 @@
+# EOFv0 for packaging legacy code in Verkle Trees
+
+The design draft that proposes the use of EOF 
+for storing code in Verkle Trees.
+An alternative to the existing method of executing 
+31-byte code chunks accompanied by 1 byte of metadata.
+
+## Goal
+
```

Comment:
I'd summarise the main goal as the following:
> Have a unified way of handling chunking (by not having chunking).

With reusing basic EOF constructs, this allows a more simplified verkle implementation supporting both "eof0 legacy" and eof1.

Reply:
Secondary objective: can this result in a better "code-to-data" ratio (by avoiding chunking)?